cmd/object-handlers-common.go

```diff
@@ -319,11 +326,23 @@ func checkPreconditions(ctx context.Context, w http.ResponseWriter, r *http.Requ
 	return false
 }
 
+// checkIfUnmodifiedSince - return true if object was modified after(Including critical values) ifUnmodifiedSinceHeader time
```

Comment:
What is a "critical value"?

Reply:
> What is a "critical value"?

example: obj.Mod = '2024-08-26 02:01:01'

"critical value" => If-Modified-Since = '2024-08-26 02:01:01', in this case, headobject will return 304